packages/vuetify/src/components/VApp/VApp.ts

```diff
@@ -1,57 +0,0 @@
-// Styles
```

Comment:
For BC restore this but just have it be a dumb functional component that does nothing